ecosystem/platform/server/app/admin/users.rb

```diff
@@ -3,6 +3,14 @@
 # Copyright (c) Aptos
 # SPDX-License-Identifier: Apache-2.0
 
+def it1_profile_link(user)
+  id = ActiveRecord::Base.connection.select_all('select id from it1_profiles where user_id = $1', nil,
```

Comment:
Use the ORM?

Reply:
I specifically left the model around so that it could still be used in the admin: https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/platform/server/app/models/it1_profile.rb

Reply:
lol the assoc you removed. My god it's been a week :-P changed.